apps/hash-api/src/graph/ensure-system-graph-is-initialized/migrate-ontology-types/migrations/016-add-initial-currency-data-types.dev.migration.ts

```diff
@@ -19,9 +43,28 @@ const migrate: MigrationFunction = async ({
 
   await createSystemDataTypeIfNotExists(context, authentication, {
     dataTypeDefinition: {
+      allOf: [{ $ref: currencyDataTypes.schema.$id }],
       title: "GBP",
-      description: "An amount denominated in British pounds sterling",
+      description: "An amount denominated in British pounds sterling.",
+      type: "number",
+      label: {
+        left: "£",
+      },
+    },
+    conversions: {},
+    webShortname: "hash",
+    migrationState,
+  });
+
+  await createSystemDataTypeIfNotExists(context, authentication, {
+    dataTypeDefinition: {
+      allOf: [{ $ref: currencyDataTypes.schema.$id }],
+      title: "EUR",
+      description: "An amount denominated in Euros.",
       type: "number",
+      label: {
+        left: "€",
```

Comment:
Euros should have the label on the right side

Reply:
[Not in English](https://en.wikipedia.org/wiki/Euro_sign#Use) 😅 

I think this is an internationalization issue... 

cc @vilkinsons

Reply:
This is indeed a case of the above. Given we're developing the English (US) version of this at the moment and deferring the localizaion/internationalization, this structure is fine for now (same decision as made recently for multi-plurality).